src/modules/rlm_eap/eap.c

```diff
@@ -72,6 +72,13 @@ static char const *eap_codes[] = {
 static int _eap_module_free(eap_module_t *inst)
 {
 	/*
+	 * Check if handle is still valid. If not, type is referencing freed memory
+	 */
+
+	if (!inst->handle)
```

Comment:
I know it seems incredibly anal, but could you put it all on one line or use braces.  http://wiki.freeradius.org/contributing/coding-standards#code-style_always-collapse-bodies-where-possible